### ISSUE TITLE
Abandoned

### ISSUE THREAD

**Issue by tasmo** [renyuneyun/Easer#483]
The last technical commit is from August 2022 a9a7e9b9ca05c45c05ddae25053e115308b59bcc

The project is not active anymore. Are there any forks or successors?

**Comment by robovoice1** [comment]
https://server47.de/automation/

**Comment by landry314** [comment]
yeah and there is a beta version that is 3 years old with no follow-thru unfortunately so it is definitely abandoned
